### ISSUE TITLE
Equippable items

### ISSUE THREAD

**Issue by PitchBright** [ForgeEssentials/CustomServerItems#3]
~~Like the custom Item, and custom Tool… can we have a Custom Armour item… so that permission will take effect whenever the player is wearing the item (/permitem mode wear).~~

~~An example of how this may be used, is a custom armour item called "Boots of Icarus". The player has the ability to fly, while they wear the boots.~~

Edit by olee: This mod has **nothing** to do with any FE item permissions - it is only about custom items.

**Comment by olee** [comment]
Adding equippable items might be intersting... but it is almost never needed...
There are already special armor mods out there which can be used.

**Comment by PitchBright** [comment]
Yeah sorry about that… I worded that really poorly. What I meant was "I would use it for the purpose of adding permissions to the player, when the player wears it."

My bad. I would love to have it though, if you feel like adding it on at some point.
